engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PassPersistance.java

```diff
@@ -54,12 +55,24 @@
 @Persistable(clazz = GenericAnnotations$.class, id = 1216)
 @Persistable(clazz = ExpressionAnnotations$.class, id = 1217)
 @Persistable(clazz = FullyQualifiedNames$.class, id = 1218)
-@Persistable(clazz = AliasAnalysis$Info$Occurrence.class, id = 1261, allowInlining = false)
-@Persistable(clazz = AliasAnalysis$Info$Scope$Root.class, id = 1262, allowInlining = false)
-@Persistable(clazz = AliasAnalysis$Info$Scope$Child.class, id = 1263, allowInlining = false)
-@Persistable(clazz = AliasAnalysis$Graph$Occurrence$Use.class, id = 1264, allowInlining = false)
-@Persistable(clazz = AliasAnalysis$Graph$Occurrence$Def.class, id = 1265, allowInlining = false)
-@Persistable(clazz = AliasAnalysis$Graph$Link.class, id = 1266, allowInlining = false)
+@Persistable(clazz = Info.Occurrence.class, id = 1261, allowInlining = false)
+@Persistable(
+    clazz = org.enso.compiler.pass.analyse.alias.Info$Scope$Root.class,
+    id = 1262,
+    allowInlining = false)
+@Persistable(
+    clazz = org.enso.compiler.pass.analyse.alias.Info$Scope$Child.class,
+    id = 1263,
+    allowInlining = false)
+@Persistable(
+    clazz = org.enso.compiler.pass.analyse.alias.Graph$Occurrence$Use.class,
```

Comment:
OK, I guess we can do those later.

Reply:
It was problematic to move them out of the same file because of `sealed` class rules Scala has.

There must be some way but I didn't want to inflate this PR  - if we _actually do need_ this changed, we can do it as separate PR.